src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -4,7 +4,10 @@
  * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
  */
 
-
+#define AXIS_COUNT 8
+#define STICK_COUNT 2
+#define BUTTON_COUNT 13
+#define INPUTERRORCHECK if (inputStateError) return false
```

Comment:
Please remove this macro and just use the `if (inputStateError) return false`... When I read a function I want to know if and when it is returning...